Taskfile.yaml

```diff
@@ -0,0 +1,78 @@
+---
+version: '3'
+
+tasks:
+  init-tools:
+    dir: tools
+    cmds:
+      - go mod tidy
+      - go mod verify
+      - go generate -x
+
+  init:
+    desc: "Install development tools"
+    deps: [init-tools]
+    cmds:
+      - go mod tidy
+
+  fmt:
+    desc: "Format code"
+    cmds:
+      - bin/gofumpt -w ./cmd/ ./internal/
+
+  env-up:
+    desc: "Start environment"
+    deps: [env-up-detach]
+    cmds:
+      - docker-compose logs --follow
+
+  env-up-detach:
+    desc: "Start dependencies in docker"
+    cmds:
+      - >
+        docker-compose up --always-recreate-deps
+        --force-recreate --remove-orphans --renew-anon-volumes --detach {{.DB}}
+    preconditions:
+      - test $DB
```

Comment:
Does `test` exist on Windows without WSL?

Reply:
Somehow it is working on Windows in base `cmd`